### DISCUSSION TITLE
outline title should not using "heading" styles.

### DISCUSSION THREAD

**Discussion by Junyi-99** [typst/typst#2521] [Q&A] [answered]
## Background
I have defined `heading` like this:

```typst
show heading.where(
    level: 1
) : it => block(
    width: 100%, 
    stroke: green,
    [
        #v(1in)
        #set text(28pt, weight: "bold")
        #set align(left)
        #if it.numbering != none [
            #block(stroke: orange, spacing: 0em, [Chapter #counter(heading).display()])
        ]
        #v(1em)
        #block(stroke: red, spacing: 0em, it.body)
        #v(0.5in)
    ])
```

which works like: 

<img width="396" alt="image" src="https://github.com/typst/typst/assets/14367694/feb751e2-719d-423b-b2eb-425927a09f29">

Pretty good so far.

Now, I need to generate a "List of Tables", so I write such code:

```typst
#let list_of_tables() = {
    set page(numbering: "i")
    // show outline : set heading()
    outline(
        title: heading(level: 1, outlined: true, supplement: [List of])[List of Tables],
        fill: repeat[.],
        target: figure.where(kind: table),
    )
}
```

Note: the reason why I use `heading` in `title` because I want the `List of Tables` to appear in another `outline`

## Problem

**When I try to use `outline` function to generate table of figures, I got this result**

(It seems like the `title` is inside another `heading`)

<img width="431" alt="image" src="https://github.com/typst/typst/assets/14367694/0b2e9108-0edf-4947-8d65-66575912597c">

## What I need

<img width="444" alt="image" src="https://github.com/typst/typst/assets/14367694/6af2a8cf-1ecf-4b69-9d61-0ddf7d66a693">

Here is how I write "Contents":

```typst
#let list_of_contents() = {
    set page(numbering: "i")
    outline(
        title: "Contents",
        fill: repeat[.],
        indent: 1cm
    )
}
```

### Reproduction URL

https://typst.app/project/rRwtbfzBJTwK-lPWcE-5uq

### Operating system

macOS

### Typst version

- [X] I am using the latest version of Typst

**Comment by EpicEricEE** [comment, marked as answer]
You can pass the title as a string and use a show-set rule for the outline heading:

```typ
#let list_of_tables() = {
  set page(numbering: "i")
  show outline: set heading(outlined: true)
  outline(
    title: "List of Tables",
    fill: repeat[.],
    target: figure.where(kind: table),
  )
}
```

**Reply by Junyi-99** [reply]
Thanks for the quick reply, problem solved!

<img width="453" alt="image" src="https://github.com/typst/typst/assets/14367694/c3a976ec-9f0c-4118-8e40-a02bad3e1234">
